tests/__init__.py

```diff
@@ -54,6 +54,25 @@ def test_multibyte_characters(self):
             "'🐍'"
         )
 
+    def test_node_child_by_field_id(self):
+        parser = Parser()
+        parser.set_language(PYTHON)
+        tree = parser.parse(b"def foo():\n  bar()")
+        root_node = tree.root_node
+        fn_node = tree.root_node.children[0]
+
+        try:
+            root_node.child_by_field_id('')
+            assert False
+        except TypeError:
+            pass
+        self.assertEqual(root_node.child_by_field_id(1), None)
+        self.assertEqual(root_node.child_by_field_id(16), None)
```

Comment:
The hard-coded `1` and `16` numbers in the tests may break if we make small changes to the python grammar. Would you be willing to *also* add a `language.field_id_for_name()` method also, so that you could write this?

```py
name_field_id = PYTHON.field_id_for_name("name")
self.assertEqual(fn_node.child_by_field_id(name_field_id).type, 'identifier')
```

Reply:
Good point (I was aware of it, but a bit too lazy to do it initially...)

Implemented.